trojsten/people/forms.py

```diff
@@ -72,7 +72,8 @@ def __init__(self, *args, **kwargs):
             label=_('Gender'),
             choices=User.GENDER_CHOICES,
         )
-        self.fields['school'].initial = 1
+        self.fields['school'].initial = None
+        self.fields['school'].empty_label = _('Other school')
```

Comment:
Z nejakého dôvodu, aj keď u seba spustím compilemessages, ostané texty (napr. helptext) sa preložia, ale toto zostáva stále Other school.... Dokonca som string "Iná škola" našiel v django.mo len raz.

Problém to bol hlavne keď som si chcel zmeniť svoju nastavenu školu na Iná škola, no to sa nedalo vybrať ... lebo v zozname bolo len Other school

Reply:
Hej, vidim to aj ja.

Reply:
Django ignouje fuzzy stringy. Fixnute.